### ISSUE TITLE
AttributeError: 'dict_items' object has no attribute 'append'

### ISSUE THREAD

**Issue by Lucas-C** [stardust85/repositorytools#27]
Hello,

There are the reproduction steps:
```
$ wget https://pypi.python.org/packages/c8/22/9460e311f340cb62d26a38c419b1381b8593b0bb6b5d1f056938b086d362/lockfile-0.12.2-py2.py3-none-any.whl

$ artifact upload lockfile-0.12.2-py2.py3-none-any.whl pip com.vsct.pip --artifact lockfile --version 0.12.2
INFO:/home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/bin/artifact:Started /home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/bin/artifact, with arguments ['upload', 'lockfile-0.12.2-py2.py3-none-any.whl', 'pip', 'com.vsct.pip', '--artifact', 'lockfile', '--version', '0.12.2']
INFO:repositorytools.lib.repository:-> Uploading lockfile-0.12.2-py2.py3-none-any.whl
Traceback (most recent call last):
  File "/home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/bin/artifact", line 11, in <module>
    sys.exit(artifact_cli())
  File "/home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/lib/python3.4/site-packages/repositorytools/cli/common.py", line 58, in __call__
    self.run(*args)
  File "/home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/lib/python3.4/site-packages/repositorytools/cli/common.py", line 55, in run
    return args_namespace.func(args_namespace)
  File "/home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/lib/python3.4/site-packages/repositorytools/cli/commands/artifact.py", line 94, in upload
    return self.repository.upload_artifacts([artifact], args.repo_id, use_direct_put=args.use_direct_put)
  File "/home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/lib/python3.4/site-packages/repositorytools/lib/repository.py", line 109, in upload_artifacts
    use_direct_put=use_direct_put)
  File "/home/lucas_cimon/.local/share/virtualenvs/nexus_uploader/lib/python3.4/site-packages/repositorytools/lib/repository.py", line 143, in _upload_artifact
    data_list.append( ('file', (filename, f, 'text/plain') ))
AttributeError: 'dict_items' object has no attribute 'append'
```

**Comment by stardust85** [comment]
Ah, the current implementation of this part of code works only for python 2.x and is not covered by unit tests (shame).

Thanks for catching it!

**Comment by stardust85** [comment]
Released to pypi as 4.2.2
